### PR TITLE
[build] Switch to netstandard based xunit

### PR DESCRIPTION
**Pull request by marek-safar**
/cc @baulig

### CODE REVIEW

**Comment by marek-safar** [comment]
@monojenkins build failed

**Comment by baulig** [comment]
The Windows failure is unrelated: `16:31:43 error CS2001: Source file 'D:\j\workspace\x\mcs\build/common/basic-profile-check.cs' could not be found.`

File: mcs/build/tests.make [hunk 1]
```diff
@@ -308,9 +308,9 @@ check: run-xunit-test-local
 xunit-test-local: $(xunit_test_lib)
 run-xunit-test-local: run-xunit-test-lib
 
-# cp -rf is a HACK for xunit runner to require xunit.execution.desktop.dll file in local folder on .net only
+# cp -rf is a HACK for xunit runner to require xunit.execution.dOTNET.dll file in local folder on .net only
 run-xunit-test-lib: xunit-test-local $(XTEST_REMOTE_EXECUTOR)
-	@cp -rf $(XTEST_HARNESS_PATH)/xunit.execution.desktop.dll xunit.execution.desktop.dll
+	@cp -rf $(XTEST_HARNESS_PATH)/xunit.execution.dotnet.dll xunit.execution.dotnet.dll
```
Comment (akoeplinger) [inline mcs/build/tests.make]: @marek-safar a few lines below we still do `@rm -f xunit.execution.desktop.dll`.

can we delete xunit.execution.desktop.dll and xunit.runner.utility.net452.dll from the xunit-binaries submodule?

Comment (marek-safar) [reply mcs/build/tests.make]: Will fix the name mismatch but we still need ` xunit.runner.utility.net452.dll` because it's the console runner dependency

**Pull request merged**